src/resources/postcss/datatables.pcss

```diff
@@ -0,0 +1,435 @@
+/*
+ * Table styles
+ */
+table.dataTable {
+  border-collapse: separate;
```

Comment:
Spacing is off a little here - should be a full tab instead of two spaces (here and the next few lines).